ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -112,203 +112,302 @@ <h3 class="title">
-            <td mat-cell *matCellDef="let element">
-              {{ element.get('srcColName').value }}
-            </td>
-          </ng-container>
-
-          <ng-container matColumnDef="srcDataType">
-            <th mat-header-cell class="table_header" *matHeaderCellDef>Type</th>
-            <td mat-cell *matCellDef="let element">
-              {{ element.get('srcDataType').value }}
-            </td>
-          </ng-container>
-
-          <ng-container matColumnDef="srcIsPk">
-            <th class="table_header" mat-header-cell *matHeaderCellDef>Pk</th>
-            <td mat-cell *matCellDef="let element">
-              <div *ngIf="element.get('srcIsPk').value">
-                <svg
-                  width="14"
-                  height="8"
-                  viewBox="0 0 14 8"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
-                >
-                  <path
-                    fill-rule="evenodd"
-                    clip-rule="evenodd"
-                    d="M4 8C5.86384 8 7.42994 6.72523 7.87398 5H10V7H13V5H14V3H7.87398C7.42994 1.27477 5.86384 0 4 0C1.79086 0 0 1.79086 0 4C0 6.20914 1.79086 8 4 8ZM6 4C6 5.10457 5.10457 6 4 6C2.89543 6 2 5.10457 2 4C2 2.89543 2.89543 2 4 2C5.10457 2 6 2.89543 6 4Z"
-                    fill="black"
+              </td>
+            </ng-container>
+
+            <ng-container matColumnDef="srcIsNotNull">
+              <th mat-header-cell class="table_header" *matHeaderCellDef>Nullable</th>
+              <td mat-cell *matCellDef="let element">
+                {{ element.get('srcIsNotNull').value ? 'Not Null' : '' }}
+              </td>
+            </ng-container>
+
+            <tr mat-header-row *matHeaderRowDef="['srcDatabase']"></tr>
+            <tr mat-header-row *matHeaderRowDef="srcDisplayedColumns"></tr>
+            <tr
+              mat-row
+              [ngClass]="{ 'scr-column-data-edit-mode': isEditMode }"
+              *matRowDef="let row; columns: srcDisplayedColumns"
+            ></tr>
+          </table>
+
+          <table mat-table [dataSource]="spDataSource">
+            <ng-container matColumnDef="spDatabase">
+              <th mat-header-cell *matHeaderCellDef class="db_name" colspan="6">
+                <div class="spanner_edit-button">
+                  <span>Spanner</span>
+                  <div *ngIf="!isEditMode && !currentObject.isDeleted">
+                    <button
+                      mat-stroked-button
+                      color="primary"
+                      (click)="toggleEdit()"
+                      *ngIf="currentObject!.isSpannerNode"
+                    >
+                      <mat-icon class="edit-icon">edit</mat-icon>
+                      EDIT
+                    </button>
+                  </div>
+                </div>
+              </th>
+            </ng-container>
+
+            <!-- <ng-container matColumnDef="srcDataType">
```

Comment:
Remove the commented code if its not required

Reply:
Removed